test/e2e/operator_test.go

```diff
@@ -2772,6 +2843,13 @@ func newHostNetworkController(name types.NamespacedName, domain string) *operato
 			},
 		},
 	}
+	if strategy != nil {
+		ig.Spec.EndpointPublishingStrategy.HostNetwork = strategy
+	}
+	if placement != nil {
+		ig.Spec.NodePlacement = placement
+	}
```

Comment:
The nil checks look superfluous.
```suggestion
	ig.Spec.EndpointPublishingStrategy.HostNetwork = strategy
	ig.Spec.NodePlacement = placement
```